test/test_permissions_api.js

```diff
@@ -0,0 +1,52 @@
+import should from 'should';
+import supertest from 'supertest';
```

Comment:
maybe you can use `supertest-as-promised` in this new file, so that you don't need to use the `done` callback :)